src/sentry/static/sentry/app/views/organizationDiscover/styles.jsx

```diff
@@ -144,6 +145,19 @@ export const ChartNote = styled(Box)`
   margin-bottom: ${space(3)};
 `;
 
+export const EditableName = styled(({value, ...props}) => (
+  <TextField value={value} inputStyle={{height: 24, padding: 2}} {...props} />
```

Comment:
Why do you need to destructure `value` if it is being turned into a prop with the same name?

Reply:
My bad, fixed